CHANGES.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## v0.2.6
+
+* Update acos-client dependency to 0.4.6
```

Comment:
I think you should write the version information from where dependent library is updated in case that user what to know the acos-client version that previous acos pack depends on, as below.
* Update acos-client dependency to 0.4.6 from 0.4.1

Reply:
Make a PR on this PR please. I have about a hundred more packs to update to Python 3.